src/Owls/parser/FoamDict.py

```diff
@@ -123,6 +124,12 @@ def footer(self):
     def separator(self):
         return "// " + "* " * 26 + "//"
 
+    def get(self, key):
+        try:
+            return self._dict[key]
+        except KeyError as e:
+            print(e)
+
```

Comment:
```suggestion
    def get(self, key, default=None):
            return self._dict.get(key, default)
```
I think it is better to just forward get to self._dict

Reply:
I did not know there was a default option for `dict.get()`. Good to know, thanks!